track.py

```diff
@@ -370,11 +409,15 @@ def crop(dfIn):
 
         self.speedLimits = crop(self.speedLimits)
         self.gradients = crop(self.gradients)
+        self.curvatures = crop(self.curvatures)
 
 
 if __name__ == '__main__':
 
     # Example on how to load and plot a track
 
-    track = Track(config={'id':'00_stationX_stationY'}, tUpper=1300)
+    track = Track(config={'id':'00_stationX_stationY_no_curvature'}, tUpper=1300)
```

Comment:
better keep only one json file (with curvatures) and simply delete the field of the track object if you want to test without curvatures.